std_msgs/Header header
geometry_msgs/Point position
geometry_msgs/Vector3 velocity
float64 yaw
float64 v_yaw
float64 radius_1
float64 radius_2
float64 dz
float64 position_x_diff
float64 position_y_diff
float64 position_z1_diff
float64 position_z2_diff
float64 position_yaw_diff
float64 velocity_x_diff
float64 velocity_y_diff
float64 velocity_yaw_diff
float64 r1_diff
float64 r2_diff